src/Files.App/ViewModels/UserControls/Widgets/BaseWidgetViewModel.cs

```diff
@@ -65,7 +65,7 @@ widgetCardItem.DataContext is not WidgetCardItem item ||
 			// Create a new Flyout
 			var itemContextMenuFlyout = new CommandBarFlyout()
 			{
-				Placement = FlyoutPlacementMode.Full
+				Placement = FlyoutPlacementMode.Right
```

Comment:
Does this work well when right clicking leftmost items?

Reply:
Yes

Reply:
I don't think this is an issue for the sidebar